### ISSUE TITLE
transform_from_displacement_field segfaults

### ISSUE THREAD

**Issue by rueberger** [ANTsX/ANTsPy#350]
the following cause segfaults:

`ants.transform_from_displacement_field(ants.from_numpy(np.random.randn(200, 200, 200)))`

`ants.transform_from_displacement_field(ants.from_numpy(np.random.randn(200, 200, 200, 3)))`

**Comment by stnava** [comment]
incorrect usage.   requires multi-channel input.

```
    field : ANTsImage
        deformation field as multi-channel ANTsImage
```

could add a check for this type of bad input or maybe do a conversion internally.

**Comment by rueberger** [comment]
right, I'm aware of the correct usage, just reporting the segfault
